spec/comm.mediawiki

```diff
@@ -11,15 +11,15 @@ Match notification via HTTP polling or other request interval-based methods are
 thus not suitable for the DEX system.
 Persistent, full-duplex communication is critical to minimizing communication
 latencies and wasted bandwidth.
-Websockets ([https://tools.ietf.org/html/rfc6455 &#91;4&#93;]) are chosen as the
+Websockets ([https://tools.ietf.org/html/rfc6455 &#91;3&#93;]) are chosen as the
```

Comment:
WebSockets.  It's quite a nit, but there does not seem to be a "websocket", only a "WebSocket".